### ISSUE TITLE
OpenFOAM11

### ISSUE THREAD

**Issue by BKeim-UB** [ubccr/software-layer#151]
Hi, 

Could you please install OpenFOAM version 11 (https://openfoam.org/version/11/)?

**Comment by tonykew** [comment]
I have built the latest version of OpenFOAM for which there is an easyconfig
file - version 2206 which uses sources from:
    https://sourceforge.net/projects/openfoam/files/v2206/

I don't know why the versioning numbers diverge between OpenFOAM on
sourceforge and OpenFOAM from openfoam.org

The files in the OpenFOAM 11 version are dated July 8th 2023
The files in the OpenFOAM v2206 version are dated July 24th 2022

The easyconfig file uses a custom patch for EasyBuild with OpenFOAM so it
is unlikely that I can just bump the version numbers to build a newer version
(from sourceforge) though I will try.

More info to follow...

**Comment by tonykew** [comment]
The newer version of OpenFOAM completes a build, but is missing many binaries.
I'm guessing this will need new patches for a complete build.
I have rebuilt v2206 against the just published version of ParaView, version 5.11.2
More info to follow...

**Comment by tonykew** [comment]
OpenFOAM has been published to the ccrsoft/2023.01 software release
I have tested launching the GUI from a terminal in CCR OnDemand
successfully.
Note: use the "NOTLEGACY" node feature when launching the CCR OnDemand session

```
$ module spider openfoam
--------------------------------------------------------------------------------------------------------------------------------------------------------------
  openfoam: openfoam/v2206
--------------------------------------------------------------------------------------------------------------------------------------------------------------
    Description:
      OpenFOAM is a free, open source CFD software package. OpenFOAM has an extensive range of features to solve anything from complex fluid flows involving
      chemical reactions, turbulence and heat transfer, to solid dynamics and electromagnetics.

    Properties:
      Physics libraries/apps

    You will need to load all module(s) on any one of the lines below before the "openfoam/v2206" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
$ module load gcc/11.2.0 openmpi/4.1.1 openfoam/v2206
$ paraFoam &
$ 
```
